kubernetes/deployment.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: issue-sync
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: issue-sync
+    spec:
+      containers:
+      - name: issue-sync
+        image: quay.io/coreos/issue-sync:v0.2.0
+        command: ["/bin/sh"]
+        args: ["-c", "while true; do ./issue-sync --config config.json; sleep $(ISSUE_SYNC_PERIOD); done"]
+        env:
+          - name: ISSUE_SYNC_PERIOD
+            valueFrom:
+              configMapKeyRef:
+                name: issue-sync
+                key: period
+          - name: ISSUE_SYNC_LOG_LEVEL
+            valueFrom:
+              configMapKeyRef:
+                name: issue-sync
+                key: log-level
+          - name: ISSUE_SYNC_GITHUB_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: issue-sync
+                key: github-token
+          - name: ISSUE_SYNC_JIRA_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: issue-sync
+                key: jira-token
+          - name: ISSUE_SYNC_JIRA_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: issue-sync
+                key: jira-secret
+          - name: ISSUE_SYNC_JIRA_CONSUMER_KEY
+            valueFrom:
+              secretKeyRef:
+                name: issue-sync
+                key: jira-consumer-key
+          - name: ISSUE_SYNC_JIRA_PRIVATE_KEY
```

Comment:
this is just the private key path, right? Let's rename the cli flag and environment variable to reflect that so we are not confused later.